Elements/src/Geometry/Vector3Extensions.cs

```diff
@@ -251,6 +251,27 @@ internal static Vector3 NormalFromPlanarWoundPoints(this IList<Vector3> points)
             }
             return normal.Unitized();
         }
+        
+        /// <summary>
+        /// Remove vectors at the same place
```

Comment:
`Remove vectors at the same place` => `De-duplicate a collection of Vectors, such that no two vectors in the result are within tolerance of each other.`